src/pricing/index.njk

```diff
@@ -273,54 +330,80 @@ hubspot:
-                </div>
-                <!-- Toggle-->
-                <div class="text-center my-8 md:mb-12">
-                    <div class="mainToggle">
-                        <button class="btnToggle">
-                            <span class="selector"></span>
-                            <div class="hosting">
-                                <div class="divCloud">
-                                    </span><span class="cloudText">CLOUD</span>
-                                </div>
-                                <div class="divSelfHosted">
-                                    <span class="selfHostedText">SELF HOSTED</span>
-                                </div>
-                            </div>
-                        </button>
-                    </div>
+                <div id="feature-comparison-table" class="md:py-9 mt-10 mb-6">
+                    <h2><span class="text-indigo-600">Feature</span> Comparison</h2>
                 </div>
                 <!-- Feature List -->
                 <div class="sticky">{{ featuresHtml | safe }}</div>
             </div>
-            <p class="max-w-screen-lg m-auto mt-12 text-center">
+            <p class="max-w-screen-lg m-auto mt-12 text-center md:px-9 pb-12 border-b">
                 At FlowFuse we understand that the security practises of our product
                 are important to your company. We've prepared a <a href="../product/security">security statement</a>
                 for you to understand the choices we've made.
             </p>
         </div>
-
     </div>
 </div>
-<div id="contact-us" class="w-full bg-gray-50 mt-16 pt-0">
+<!-- Social proof -->
+<div class="max-w-md sm:max-w-screen-lg m-auto max-w-5xl pb-7 md:px-9">
+    <div class="social-proof-logos mx-auto w-full text-center">
+        {% include "social-proof.njk" %}
+    </div>
+</div>
+<!-- Form section -->
+<div id="request-trial-license" class="contentSelfHosted hide w-full bg-gray-50 pt-0">
     <div class="w-full bg-gray-50 pb-12">
         <div class="container max-w-5xl m-auto grid grid-cols-1 md:grid-cols-2 px-6 pt-16 gap-x-16 gap-y-6">
             <div>
-                <h4 class="w-full text-center md:text-left">
-                    Contact Us
-                </h4>
+                <h3 class="w-full text-center md:text-left">
+                    Request a trial license
+                </h3>
                 <p>
-                    Excited to explore FlowFuse Enterprise together? Share your info and we'll get in touch.
+                    Unlock the complete potential of FlowFuse with a complimentary 30-day Enterprise license. This trial
+                    gives you the chance to explore all the features and functionalities of FlowFuse in your own
+                    environment. To start your trial, just fill out the form.
                 </p>
             </div>
             {% include hubspot.script %}
         </div>
     </div>
 </div>
 
+<!-- Frequently Asked Questions -->
```

Comment:
Isn't there a partial for FAQs? If not, we should introduce that in this PR

Reply:
There is now! 😉